components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -255,13 +263,14 @@ public void deleteUser(String userId) throws NotFoundException, CharonException
     }
 
     @Override
-    public List<Object> listUsersWithGET(Node rootNode, int startIndex, int count, String sortBy,
-                                         String sortOrder, Map<String, Boolean> requiredAttributes)
-            throws CharonException, NotImplementedException, BadRequestException {
-        if(sortBy != null || sortOrder != null) {
+    public List<Object> listUsersWithGET(Node rootNode, int startIndex, int count, String sortBy, String sortOrder,
```

Comment:
Since this is a API change, do we support backward compatibilty with listUsersWithGET old method?

Reply:
since we released this API changes with a newer vresion we don't need to maintain backward compatibility for those API?